src/store/onboarding/actions.js

```diff
@@ -190,6 +193,12 @@ export function finalizeNewAccount() {
 
     // Finally unlock the Safe (enable UI again)
     await dispatch(unlockSafeDeployment());
+    await dispatch(
+      notify({
+        text: <WelcomeMessage />,
+        type: NotificationsTypes.WARNING,
```

Comment:
TYPE should be special for welcome message

Reply:
Or warning is meant to be the special pink on white notification? in that case the styling of warning is incorrect?

Reply:
We can change type for special but I decided to stick with WARNING as changing version produce error in console.
![Screenshot from 2023-01-23 17-36-46](https://user-images.githubusercontent.com/95082155/214053806-a62815f0-cd05-458f-a4d0-42e9df030628.png)

currenty I updated it in a way that we use SPECIAL type which at the end produce 'warning' string (see notifications/actions.js) so error is avoided